packages/block-library/src/pattern/edit.js

```diff
@@ -9,7 +9,12 @@ import {
 	useBlockProps,
 } from '@wordpress/block-editor';
 
-const PatternEdit = ( { attributes, clientId } ) => {
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../lock-unlock';
+
+const PatternEdit = ( { attributes, clientId, rootClientId } ) => {
```

Comment:
The `rootClientId` will be `undefined` since the block edit component doesn't receive it as a prop.

https://github.com/WordPress/gutenberg/blob/561d2447ea5e8292ad1b05fe64cd02c374d9ef79/packages/block-editor/src/components/block-list/block.js#L124-L142

Reply:
Good catch @Mamaduka! I've opened a tiny follow-up in https://github.com/WordPress/gutenberg/pull/53206